src/plugins/presentation_util/public/components/dashboard_drilldown_options/dashboard_drilldown_options.tsx

```diff
@@ -52,6 +52,34 @@ export const DashboardDrilldownOptionsComponent = ({
             onChange={() => onOptionChange({ openInNewTab: !options.openInNewTab })}
             data-test-subj="dashboardDrillDownOptions--openInNewTab--checkbox"
           />
+          <EuiSpacer size="s" />
+          <EuiSwitch
+            compressed
+            name="overflowEllipsis"
+            label={dashboardDrilldownConfigStrings.component.getOverflowEllipsis()}
+            checked={options.overflowEllipsis}
+            onChange={() =>
+              onOptionChange({
+                overflowEllipsis: !options.overflowEllipsis,
+                overflowTextWrap: !options.overflowTextWrap,
+              })
+            }
+            data-test-subj="dashboardDrillDownOptions--overflowEllipsis--checkbox"
+          />
+          <EuiSpacer size="s" />
+          <EuiSwitch
+            compressed
+            name="overflowTextWrap"
+            label={dashboardDrilldownConfigStrings.component.getOverflowTextWrap()}
+            checked={options.overflowTextWrap}
+            onChange={() =>
+              onOptionChange({
+                overflowTextWrap: !options.overflowTextWrap,
+                overflowEllipsis: !options.overflowEllipsis,
+              })
+            }
+            data-test-subj="dashboardDrillDownOptions--overflowTextWrap--checkbox"
+          />
```

Comment:
@rshen91 I think it is confusing to have seperate "Wrap text" and "Ellipses" toggles - these should be used for entirely distinct settings, and they shouldn't be "linked" (pun intended) together like this: 

![Nov-14-2024 08-12-19](https://github.com/user-attachments/assets/3ae638da-82b7-4518-9fa7-1fe6f8b457ba)


I think a [button group](https://eui.elastic.co/#/navigation/button#buttonGroup-compressed), where **only a single option is allowed** might be a better option here. Pinging @andreadelrio here to see if she has a better suggestion, though.

Reply:
Good call! I updated the UI in [1ac9d8d](https://github.com/elastic/kibana/pull/199844/commits/1ac9d8df66a704875b065eed0bbe2c5958ab80b7). I'm open to suggestions with any labels and naming thanks!!!

Reply:
If we keep the setting per-link, I think the "Overflow" setting should come first:

| Header | Header |
|--------|--------|
| ![image](https://github.com/user-attachments/assets/1d5dc4c6-a6ec-47e2-8e6b-ea7b04d3f900) | ![image](https://github.com/user-attachments/assets/0f479f96-99ed-4212-b010-3f0702bef2a1) | 

This ties in to https://github.com/elastic/kibana/pull/199844#discussion_r1842466146, since we shouldn't be putting this setting in the shared component - it should be just for links.

I also wonder if we could do something with an icon instead? This is just an idea, though - would definitely need @andreadelrio's opinion here 😆 

![Nov-14-2024 09-09-46](https://github.com/user-attachments/assets/3de7ecb7-57e6-4b3d-92e1-5e3e2f9671ff)

Reply:
+1 to using a button group.